WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -18418,7 +18416,6 @@
 				17039226282E6D2F00F602E9 /* ViewsVisitorsLineChartCell.xib in Resources */,
 				FABB28472603067C00C8785C /* Launch Screen.storyboard in Resources */,
 				F465978F28E65F8A00D5F49A /* celadon-on-white-icon-app-60@2x.png in Resources */,
-				FEC3B81A26C2915A00A395C7 /* SingleButtonTableViewCell.xib in Resources */,
```

Comment:
The changes in this file clean up what was most likely an incorrect merge on the project file.

See 3fc0ac0a67158c180ce263208c81af8242430640.